src/components/pages/homepage/press-logos.module.scss

```diff
@@ -8,5 +8,6 @@
     filter: grayscale(1);
     height: 40px;
     margin: 1rem 2rem;
+    margin-bottom: 3rem;
```

Comment:
Thanks for this, @karinanguyen ! I think what @julialedur was hoping for in #688 was to add margin between the article titles and the press logos, where the two grey lines are in this pic:
![image](https://user-images.githubusercontent.com/18607205/80122244-82fe4f00-854a-11ea-8acf-3961fdb8c451.png)

I _think_ you could do that by applying the same patch to `.logos` (the container) instead of `img` (the children).